tests/testUtils.js

```diff
@@ -84,5 +96,10 @@ module.exports = {
     .then(() => done())
     .catch((err) => done(err));
   }, // forceDeleteUser
-
+  // delete users
+  forceDeleteSubject(done) {
+    forceDelete(db.Subject, testStartTime)
+    .then(() => done())
+    .catch((err) => done(err));
+  }, // forceDeleteUser
 }; // exports
```

Comment:
Yes there is, and we have delete subjects in tests/api/v1/subjects/utils.js. However it deletes tags, profiles, user, and aspect. The section I added only deletes subjects.